projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -1181,6 +1181,7 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
         this.onOpened.emit(this);
 
         // TODO: remove this line after deprecating 'onOpen'
```

Comment:
Please update the comment to state that the following two lines should be removed.